packages/trigger-client/src/types/clientTypes.ts

```diff
@@ -44,7 +43,7 @@ export type TriggerOrderStatus =
 interface SignatureParams {
   // Orderbook address for placement, endpoint address for cancellation & listing
   verifyingAddr: string;
-  chainId: BigNumberish;
+  chainId: BigDecimalish;
```

Comment:
Could this just be a `number`? It seems like `viem`/`wagmi` types `chainId: number`